cot/templates/admin/model.html

```diff
@@ -34,6 +34,40 @@ <h2>{{ model.name() }}</h2>
 
     <footer>
         Displaying {{ objects.len() }} {{ model.name() }}{{ objects.len()|pluralize }}.
+
+        <div class="pagination">
+            <select id="page-size-selector">
+                {% for option in [10, 20, 30, 40] %}
+                    <option value="{{ option }}" {% if option == page_size %}selected{% endif %}>{{ option }}</option>
+                {% endfor %}
+            </select>
+
+            {% if page > 1 %}
+                <a href="?page={{ page - 1 }}&page_size={{ page_size }}" class="btn">Previous</a>
```

Comment:
This way, the button will actually have a button-like appearance.

```suggestion
                <a href="?page={{ page - 1 }}&page_size={{ page_size }}" class="btn secondary">Previous</a>
```